ci/test

```diff
@@ -7,9 +7,17 @@
 # this script.  We should fix these errors and turn them on.
 set -e
 
+# TODO: Documentation of higher level dependencies.  Partial list:
+# 1. All the Azure env args assumed to exist.
+# 2. Same for OIDC / GCP.
+# 3. Why are we calling via sh in Jenkinsfile when these are bash files?
```

Comment:
It looks like it does use the shebang to determine the interpreter: https://www.jenkins.io/doc/pipeline/steps/workflow-durable-task-step/#sh-shell-script